### ISSUE TITLE
Add https://www.htmlelements.com/

### ISSUE THREAD

**Issue by bmarkov** [webcomponents/custom-elements-everywhere#559]
Smart HTML Elements for building Custom Elements: https://www.htmlelements.com/

**Comment by rictic** [comment]
The site looks like it's now a site selling custom elements (cool!) rather than a library for rendering HTML. Is that the case?

Also do you have usage numbers we could see? The design of the page currently means that we need to limit the list to libraries that are at least somewhat widely used, otherwise the info that people are most likely coming to see will become steadily harder and harder to find. I'd like to add better filtering and sorting at some point, but for the moment we're looking for some rough gage that each library has at least moderately wide uptake in the open source world. If this isn't there yet we can always reevaluate in a few months

**Comment by bmarkov** [comment]
> The site looks like it's now a site selling custom elements (cool!) rather than a library for rendering HTML. Is that the case?
> 
> Also do you have usage numbers we could see? The design of the page currently means that we need to limit the list to libraries that are at least somewhat widely used, otherwise the info that people are most likely coming to see will become steadily harder and harder to find. I'd like to add better filtering and sorting at some point, but for the moment we're looking for some rough gage that each library has at least moderately wide uptake in the open source world. If this isn't there yet we can always reevaluate in a few months

Yes, we built a commercial components library with our rendering APIs. About the usage numbers - we have ~200 daily downloads of the library. Not sure how many continue using it after trying it.
